scripts/admincommands.js

```diff
@@ -1,9 +1,8 @@
-exports.handleCommand = function(src, command, commandData, tar, channel) {
-    if (command === "memorydump") {
-        sys.sendMessage(src, sys.memoryDump(), channel);

```

Comment:
Oak also can't code. Commenting is better just in case someone ever decides to fix it, or still uses it on other servers. Doesn't hurt anything to be commented out.